engines/grim/emi/modelemi.cpp

```diff
@@ -79,9 +79,9 @@ void EMIMeshFace::loadFace(Common::SeekableReadStream *data) {
 			y = data->readSint32LE();
 			z = data->readSint32LE();
 		} else {
-			x = data->readSint16LE();
-			y = data->readSint16LE();
-			z = data->readSint16LE();
+			x = data->readUint16LE();
```

Comment:
This can't be negative?

Reply:
I assumed no. 

I'm not a specialist of OpenGL but the only types supported for indices in glDrawElements seem GL_UNSIGNED_BYTE, GL_UNSIGNED_SHORT or GL_UNSIGNED_INT (depending on the version).
I wonder if the same modification cannot be done for PS2 version too.

Reply:
`_indexes` are indices into a larger array of vertices, so cannot be negative.

Reply:
Thank you for you confirmation Botje. So, I can make the same modification for PS2 version ?

Reply:
Yep!

Reply:
Done and committed.